Dockerfile

```
@@ -0,0 +1,6 @@
+FROM australia-southeast1-docker.pkg.dev/analysis-runner/images/driver:62e79e54d43aa1762ec68a6661872320a39517bd-hail-2ea2615a797a5aff72d20d9d12a2609342846a07
+
+ADD README.md .
+ADD cpg_utils .
+ADD setup.py .
```

`COPY` is generally preferable to `ADD`: https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#add-or-copy

```suggestion
COPY README.md .
COPY cpg_utils .
COPY setup.py .
```